test/cpp/jit/test_misc.h

```diff
@@ -643,6 +644,23 @@ void testNoneSchemaMatch() {
   // checking that constant propagation ran wo/failure
   AT_ASSERT(std::distance(nodes.begin(), nodes.end()) == 1);
 }
+
+void fakePass(std::shared_ptr<Graph>&g) {
+  return;
+}
+
+RegisterPass p(fakePass);
+
+void testPassManagement() {
```

Comment:
We should also have a test that the passes actually run when using the executor.